keps/sig-api-machinery/2161-apiserver-default-labels/README.md

```diff
@@ -253,18 +244,20 @@ Since this has been reserved since 2017 (if not earlier), no backwards guarantee
 
 1.22:
 - gate enabled by default
-- remove gate
 - release note
 
+1.23:
+- remove the gate
+
 ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Yes, network policies which are already using this label, may break, although such policies would be in violation of the standard that k8s.io labels are reserved.
+  Yes, namespaces which are already targetted via this label may break, although such policies would be in violation of the standard that k8s.io labels are reserved.
 
 * **What specific metrics should inform a rollback?**
-  No e2e tests or metrics exist to inform this, but if a net loss of healthy pods occurred and a user is actively using networkpolicies that impersonate on the k8s.io label, that would be a hint. 
+  No e2e tests or metrics exist to inform this, but if a net loss of healthy pods occurred and a user is actively using networkpolicies that impersonate on the k8s.io label, that would be a hint.  This could be discovered by inspection of APIServer rejection events associated with the namespace API resource.
```

Comment:
indicate this metric name https://github.com/kubernetes/apiserver/blob/master/pkg/endpoints/metrics/metrics.go#L78 and the status code (I'm pretty sure it's a 422)

Also update the monitoring section of the PRR

Reply:
just added ```apiserver_request_total{code="422",component="apiserver",contentType="application/json",dry_run="",group="",resource="namespaces",scope="resource",subresource="",verb="POST",version="v1"} ``` reference